container/features/src/main/resources/features.xml

```diff
@@ -98,11 +98,11 @@
       <bundle>wrap:mvn:org.apache.xmlgraphics/batik-xml/1.7</bundle>
     </feature>
 
-    <feature name="c3p0" description="c3p0" version="0.9.1.2">
+    <feature name="c3p0" description="c3p0" version="${c3p0Version}">
       <feature>postgresql</feature>
 
       <!-- Add all possible JDBC drivers to the c3p0 import list so that it can classload them -->
-      <bundle>wrap:mvn:c3p0/c3p0/0.9.1.2$Bundle-SymbolicName=c3p0&amp;Bundle-Version=0.9.1.2&amp;Import-Package=javax.management,javax.naming,javax.naming.spi,javax.sql,javax.xml.parsers,org.apache.log4j,org.postgresql,org.w3c.dom</bundle>
+      <bundle>wrap:mvn:com.mchange/c3p0/${c3p0Version}$Bundle-SymbolicName=c3p0&amp;Bundle-Version=${c3p0Version}&amp;Import-Package=javax.management,javax.naming,javax.naming.spi,javax.sql,javax.xml.parsers,org.apache.log4j,org.postgresql,org.w3c.dom</bundle>
```

Comment:
I think that you can update the Import-Package section here:

* Remove javax.naming.spi
* Remove org.apache.log4j

Can you make these changes and make sure that the Karaf integration tests work before merging this? Go into opennms-full-assembly and get rid of the @Ignore annotations temporarily and then run the tests as usual (mvn -DskipITs=false clean install). Thanks!